charts/incubator/minecraft-bungeecord/questions.yaml

```diff
@@ -313,7 +312,7 @@ questions:
                                   description: "The internal(!) port on the container the Application runs on"
                                   schema:
                                     type: int
-                                    default: 25575
+                                    default: 25578
```

Comment:
You can't use same `port` or `targetPort` on 2 different services of the same app